microsoftgraph/microsoft-graph-explorer-v4
# Task: Add Graph Support Information to README.md

## Overview

Closes #2033 

### Demo

![image](https://user-images.githubusercontent.com/34104277/212283348-907d5fa6-cea7-417c-92ed-63773fd55d09.png)

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout to this branch.
* View README.md file.
* Observe additions under "Getting Help & Guides" and test links.
